applications/source/zeromq-source/README.adoc

```diff
@@ -0,0 +1,49 @@
+//tag::ref-doc[]
+= ZeroMQ Source
+
+The "zeromq" source enables receiving messages from ZeroMQ.
+
+== Input
+
+N/A
+
+== Output
+
+=== Payload
+
+* `byte[]`
+
+== Options
+
+The **$$zeromq$$** $$source$$ has the following options:
+
+//tag::configuration-properties[]
+$$zeromq.supplier.bind-port$$:: $$Bind Port for connecting to the ZeroMQ TCP Server.$$ *($$Integer$$, default: `$$0$$`)*
+$$zeromq.supplier.connect-url$$:: $$Connection URL for to the ZeroMQ TCP Server.$$ *($$String$$, default: `$$<none>$$`)*
+$$zeromq.supplier.consume-delay$$:: $$The delay to consume from the ZeroMQ TCP Server when no data received.$$ *($$Duration$$, default: `$$1s$$`)*
+$$zeromq.supplier.socket-type$$:: $$The {@link SocketType} the connection should make.$$ *($$SocketType$$, default: `$$<none>$$`)*
```

Comment:
Remember I told you to not use JavaDocs tag in those configuration properties description? 😉